views/editor/nav-options.html

```diff
@@ -77,6 +77,16 @@
                   <img id="sun-white" class="hide" src="/img/icon/theme-sun-white.svg" width="20px" class="hide" height="20px">
                   <img id="sun-green" src="/img/icon/theme-sun-green.svg" width="20px" height="20px">
                 </div>
+              </li>
+              <li>
+                <span>SVG/XML</span>
```

Comment:
This should be a string in the locale/en-US messages.properties file, fetched with the same `{{ gettext(......) }}` calls you see above/below these lines